src/TuringTutorials.jl

```diff
@@ -91,23 +92,23 @@ function weave(
 end
 
 # Weave all tutorials
-function weave(; kwargs...)
-    for folder in readdir(joinpath(REPO_DIR, "tutorials"))
-        weave(folder; kwargs...)
+function weave(source_folder::AbstractString; kwargs...)
```

Comment:
I think we still need a default no-argument `weave` function that processes all docs and tutorials.

Reply:
Done